### PR TITLE
Add isort

### PR DESCRIPTION
**Pull request by daviddavis**
[noissue]

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -8,3 +8,8 @@ issue_format = "`#{issue} <https://github.com/pulp/pulp-cli/issues/{issue}>`_"
 
 [tool.black]
 line-length = 100
+
+[tool.isort]
+multi_line_output = 3
+include_trailing_comma = true
```
Comment (mdellweg) [inline pyproject.toml:14]:
```suggestion
profile = "black"
multi_line_output = 3
```

The docs say, you should use something like this.

**Review by mdellweg — APPROVED** [review]
I was asking myself before, whether we should move the linting from the matrixed test workflows to the single check workflow. WDYT?

**Comment by daviddavis** [comment]
> I was asking myself before, whether we should move the linting from the matrixed test workflows to the single check workflow. WDYT?

I like the idea of having a separate workflow for linting. What do you mean by "single check"? I kind of like how we use different steps for different linters though as it makes it easy to scan a job and see what's failing.

**Comment by mdellweg** [comment]
> > I was asking myself before, whether we should move the linting from the matrixed test workflows to the single check workflow. WDYT?
> 
> I like the idea of having a separate workflow for linting. What do you mean by "single check"? I kind of like how we use different steps for different linters though as it makes it easy to scan a job and see what's failing.

I meant the "issue checker" workflow.

**Pull request merged**